### PR TITLE
Remove deprecated each function

### PR DESCRIPTION
**Pull request by pierredup**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | 

Replace the `each()` function which is deprecated in PHP 7.2 (https://wiki.php.net/rfc/deprecations_php_7_2#each)

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php [hunk 1]
```diff
@@ -170,9 +170,11 @@ public function testGlobalsWithDifferentTypesAndValues()
 
         $calls = $container->getDefinition('twig')->getMethodCalls();
         foreach (array_slice($calls, 1) as $call) {
-            list($name, $value) = each($globals);
+            list($name, $value) = array(key($globals), current($globals));
```
Comment (Tobion) [inline src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php]: IMO you can just remove the whole line and use `key` and `current` in place of the variables directly.

**Review by smithandre — APPROVED** [review]

**Comment by xabbuh** [comment]
Is this the only place in the code where `each()` is being used?

**Comment by pierredup** [comment]
Yes, it's the only place I could find

**Review by xabbuh — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @pierredup.

**Pull request closed without merging**